docs/repair_based_node_ops.md

```diff
@@ -0,0 +1,60 @@
+# Repair based node operations
+
+Here below is a simple introduction to the node operations Scylla supports and
+some of the issues.
+
+- Replace operation
+
+   It is used to replace a dead node. The token ring does not change. Replacing
+   node pulls data from only one of the replicas which might not be the latest
+   copy.
+
+- Rebuild operation
+
+   It is used to get all the data this node owns form other existing nodes. It
+   pulls data from only one of the replicas which might not be the latest copy.
+
```

Comment:
Q: does rebuild drop the rebuilding replica's data before rebuild?

Reply:
No, it does not.